packages/admin/cms-admin/src/generator/future/__tests__/grid/queryParamsPrefix.test.ts

```diff
@@ -0,0 +1,41 @@
+import { buildSchema, introspectionFromSchema } from "graphql";
+
+import { generateGrid } from "../../generateGrid";
+import { GridConfig } from "../../generator";
+
+describe("Admin Generator", () => {
+    describe("Grid", () => {
+        it("should generate correct query params prefix", async () => {
+            const schema = buildSchema(`
+                type Query {
+                    product(id: ID!): Product!
```

Comment:
This isn't what the API Generator generates. We should discuss if and how we can re-use the API Generator output here.